ui/src/api-logic/mappers/syncs-logs.ts

```diff
@@ -0,0 +1,58 @@
+import { format, lightFormat } from 'date-fns'
+import { SyncLogsType, SyncTypeData } from 'src/@types'
+import { getSimpleDurationTime, mapToRepoSyncStateT } from 'src/utils'
+import { DATE_FORMAT, GITHUB_URL } from 'src/utils/constants'
+import { GetSyncHistoryLogsQuery } from '../graphql/generated/schema'
+
+/**
+ * Method which iterate each sync and map it to SyncTypeData to be shown in table 
+ * @param data Sync list that comes from data base in GetSyncHistoryLogsQuery format
+ * @returns SyncTypeData with syncs info and its SyncLogsType info by each sync
+ */
+const mapToSyncLogsData = (data: GetSyncHistoryLogsQuery | undefined): SyncTypeData => {
+  // General repo info 
+  const repoData: SyncTypeData = {
+    repo: {
+      id: data?.repo?.id,
+      name: data?.repo?.repo.replace(GITHUB_URL, '') || '',
+      type: data?.repo?.isGithub ? 'github' : 'other',
+    }
+  }
+
+  // 1. Get Sync general info
+  data?.repo?.repoSyncs.nodes.forEach((s) => {
+    const syncInfo = {
+      id: s.id,
+      title: s?.syncType.replaceAll('_', ' ') || '',
+      brief: s?.repoSyncTypeBySyncType?.description || '',
+      syncState: s?.repoSyncQueues.nodes.length !== 0 ? mapToRepoSyncStateT(s?.repoSyncQueues.nodes[0]?.status || '') : 'empty',
+    }
+    repoData.sync = syncInfo
```

Comment:
```suggestion
    repoData.sync = {
      id: s.id,
      title: s?.syncType.replaceAll('_', ' ') || '',
      brief: s?.repoSyncTypeBySyncType?.description || '',
      syncState: s?.repoSyncQueues.nodes.length !== 0 ? mapToRepoSyncStateT(s?.repoSyncQueues.nodes[0]?.status || '') : 'empty',
    }
```

Reply:
I'm gonna merge this branch, and gonna fix this point in next PR :)

Reply:
Done [cf5ddad](https://github.com/mergestat/fuse/pull/118/commits/cf5ddadcfbec46a96bea8a2e89f00252cdd0f448)